packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -52,6 +55,8 @@ export const PaymentConfirmation = ({
     interval_count
   );
 
+  const { l10n } = useLocalization();
```

Comment:
I don't need to call this all the time; only if `buttonLabel` is falsy. Is there a way this can be done lazily?

Reply:
I like your line of thinking here, but I don't think we need to worry about it.  I suspect that that call isn't that expensive.